.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=4883116)